### PR TITLE
Updating `v3.4` folder content with `next` folder content

### LINKED ISSUE #163: Copy `next` into `v3.4`
Based on the conversation in https://github.com/etcd-io/website/issues/152, https://github.com/etcd-io/website/issues/155, and https://github.com/etcd-io/website/pull/161:

Should we copy `next` (previously `current`) into `v3.4` now? `next` has the updated descriptions that were introduced when we migrated the docs over from etcd-io/etcd, and I think `next` best represents the latest `v3.4` version.

_Originally posted by @nate-double-u in https://github.com/etcd-io/website/issues/161#issuecomment-807427766_

### PR DESCRIPTION
**Pull request by nate-double-u**
This PR pulls in the content of the `next` folder into the `v3.4` folder, while not copying parts of the `next` folder that pertain to the upcoming `v3.5` release.

Closes #163

/cc @spzala @ptabor @chalin

### CODE REVIEW

**Comment by nate-double-u** [comment]
deploy preview: https://deploy-preview-169--etcd.netlify.app/docs/v3.4/

There are several pages that I'd like to request particular attention for (@ptabor, maybe you could do this? I think you've got a good understanding of what's coming up in v3.5)

Should the following be brought into `v3.4` (currently not)
* https://deploy-preview-169--etcd.netlify.app/docs/next/dev-internal/modules

Do these pages have any v3.5 content that I missed?
*  https://deploy-preview-169--etcd.netlify.app/docs/v3.4/dev-guide/api_concurrency_reference_v3/
* https://deploy-preview-169--etcd.netlify.app/docs/v3.4/dev-guide/api_grpc_gateway/
* https://deploy-preview-169--etcd.netlify.app/docs/v3.4/dev-guide/api_reference_v3/
* https://deploy-preview-169--etcd.netlify.app/docs/v3.4/dev-guide/grpc_naming/
* https://deploy-preview-169--etcd.netlify.app/docs/v3.4/learning/api_guarantees/
* https://deploy-preview-169--etcd.netlify.app/docs/v3.4/learning/why/
* https://deploy-preview-169--etcd.netlify.app/docs/v3.4/op-guide/configuration/#--auth-token-ttl

**Comment by chalin** [comment]
With the doc versions specified as vX.Y now, thankfully, this PR will be the last such retrograde update (if the PR does end up landing).

File: content/docs/v3.4/faq.md [hunk 1]
```diff
@@ -74,7 +82,7 @@ etcdctl provides a `snapshot` command to create backups. See [backup][backup] fo
 
 When replacing an etcd node, it's important  to remove the member first and then add its replacement.
 
-etcd employs distributed consensus based on a quorum model; (n/2)+1 members, a majority, must agree on a proposal before it can be committed to the cluster. These proposals include key-value updates and membership changes. This model totally avoids any possibility of split brain inconsistency. The consequence of permanent quorum loss is catastrophic.
+etcd employs distributed consensus based on a quorum model; (n/2)+1 members, a majority, must agree on a proposal before it can be committed to the cluster. These proposals include key-value updates and membership changes. This model totally avoids any possibility of split brain inconsistency. The downside is permanent quorum loss is catastrophic.
```
Comment (spzala) [inline content/docs/v3.4/faq.md:85]: This modified sentence is incorrect - I would keep the original sentence.

File: content/docs/v3.4/op-guide/clustering.md [hunk 2]
```diff
@@ -268,7 +268,7 @@ This will cause each member to register itself with the custom etcd discovery se
 
 #### Public etcd discovery service
 
-If no existing cluster is available, use the public discovery service hosted at `discovery.etcd.io`.  To create a private discovery URL using the "new" endpoint, use the command:
+If no exiting cluster is available, use the public discovery service hosted at `discovery.etcd.io`.  To create a private discovery URL using the "new" endpoint, use the command:
```
Comment (spzala) [inline content/docs/v3.4/op-guide/clustering.md:271]: why this change?

**Review by spzala — COMMENTED** [review]
Thanks @nate-double-u - great work as usual but can we split this PR into small PRs with a exact reason for changes for a better review? I have added couple of comments but smaller PRs would make it easier to review. Also, why are we moving content of "next" to the 3.4?

File: content/docs/v3.4/op-guide/clustering.md (hunk 2 above)
Comment (nate-double-u) [reply content/docs/v3.4/op-guide/clustering.md:271]: Sorry, good catch, I'll fix this. This is an artifact of merging in from the `next` directory. I'll fix it there too.

File: content/docs/v3.4/faq.md (hunk 1 above)
Comment (nate-double-u) [reply content/docs/v3.4/faq.md:85]: I'll make a note of that, and see if it needs correcting in the `next` directory as well.

**Comment by nate-double-u** [comment]
> ... can we split this PR into small PRs with a exact reason for changes for a better review? 

I'll see what I can do -- I agree, larger PRs are harder to review.

> ... why are we moving content of "next" to the 3.4?

I think that we need to do this because we have moved from `current` to `next`, meaning that some of what was put into `current` should be migrated into the new `v3.4` folder. Before we [shifted our release scheme](https://github.com/etcd-io/website/issues/152) the `v3.4` folder was `v3.4.0`, which fell a little bit behind when the assumption was that `current` would be made into `v3.4.X` for the next release.
